cartography/intel/okta/test.py

```diff
@@ -0,0 +1,796 @@
+# Okta intel module
+import logging
+import json
+from okta import UsersClient, UserGroupsClient, AppInstanceClient, FactorsClient
```

Comment:
I recommend splitting this file up by client type.  For example, assets that are returned by the `FactorsClient` should be under `cartography/intel/okta/factors.py`, and assets from the `UsersClient` should be under `cartography/intel/okta/users.py`